src/graphs/algorithms/course-schedule-1.ts

```diff
@@ -0,0 +1,37 @@
+import Queue from '../../queue/queue.ts';
```

Comment:
@mandy8055,
Please add the problem link in the comments here as well.

Reply:
@ManujJiraUser,
Sure I am planning to raise a PR for updating all the missing pieces which I feel. This will also be factored in there.